### PR TITLE
[SCIP] change SHA for new version

### PR DESCRIPTION
**Pull request by matbesancon**
Same as https://github.com/JuliaPackaging/Yggdrasil/pull/4169 for SCIP alone

### CODE REVIEW

**Comment by matbesancon** [comment]
is there a problem with GMP on aarch / arm?

**Comment by giordano** [comment]
You have to use GMP 6.2 (or 6.2.1, I don't remember, search the repo) for the new architectures

**Comment by giordano** [comment]
Latest version of `bliss_jll` generated a binary file `${prefix}/bin`, which conflicts with the `${bindir}`.  Sigh

**Comment by matbesancon** [comment]
Oh thanks I would not have found it. Should we fix / remove the created binary in the bliss build?

**Comment by giordano** [comment]
#4197

**Comment by giordano** [comment]
For the record, `cp` has a weird behaviour for which it does different things depending on whether the target is an existing directory or not: if it isn't an existing directory, it creates a file with the given name, otherwise it creates a file _inside_ that directory.  For this reason I generally prefer to do something like `cp foo destdir/.`, or use `install -D`.  Using `cp foo destdir` is a guaranteed footgun

File: S/SCIP/build_tarballs.jl [hunk 1]
```diff
@@ -52,7 +52,7 @@ products = [
 dependencies = [
     Dependency(PackageSpec(name="bliss_jll", uuid="508c9074-7a14-5c94-9582-3d4bc1871065")),
     Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae")),
-    Dependency(PackageSpec(name="GMP_jll", uuid="781609d7-10c4-51f6-84f2-b8444358ff6d"), v"6.1.2"),
+    Dependency(PackageSpec(name="GMP_jll", uuid="781609d7-10c4-51f6-84f2-b8444358ff6d"), v"6.2.1"),
```
Comment (giordano) [inline S/SCIP/build_tarballs.jl:55]: It was 6.2.0 the right one
```suggestion
    Dependency(PackageSpec(name="GMP_jll", uuid="781609d7-10c4-51f6-84f2-b8444358ff6d"), v"6.2.0"),
```

**Pull request merged**